README.md

```diff
@@ -52,26 +53,65 @@ const store = createStore(
 
 ## 🎛️ API
 
-### `reducers`
+### Overview
 
-The default export is an object of reducers meant to be included in a call to
-Redux's [`combineReducers`](https://redux.js.org/api/combinereducers) when
-setting up the store. Included reducers are `browser`, `error`, `errorReporting`, `intl`,`redirection` and `rendering`.
+[Ducks](https://github.com/erikras/ducks-modular-redux) include the reducer, selectors and action
+creators for a given branch of state. These ducks make up the core of One App's state.
+
+- [`browser`](#browser-duck)
+- [`error`](#error-duck)
+- [`errorReporting`](#errorreporting-duck)
+- [`intl`](#intl-duck)
+- [`redirection`](#redirection-duck)
+- [`rendering`](#rendering-duck)
+
+### Reducers
+
+The default export is an object of reducers meant to be included in a call to Redux immutable's
+[`combineReducers`](https://redux.js.org/api/combinereducers) when setting up the store. Included
+reducers are `browser`, `error`, `errorReporting`, `intl`, `redirection` and `rendering`.
 
 ```js
-import { combineReducers } from 'redux';
+import { combineReducers } from 'redux-immutable';
 import globalReducers from '@americanexpress/one-app-ducks';
 
 const appReducer = combineReducers({
   ...globalReducers,
-  ...otherReducers, // Your specific reducers
+  ...otherReducers,
 });
+```
+
+The state shape follows the schema below:
+
+### `browser` duck
+
+This duck is for reading information about the user's browser. It is particularly helpful on the
+server.
+
+- [State shape](#state-shape)
+- [Selectors](#selectors)
+- [Action creators](#action-creators)
```

Comment:
What about structuring table of contents like...

```
- [State shape](#state-shape)
- Selectors
  - [`getCookies`](#get-cookies)
- Action creators
  - [`setUserAgent`](#setuseragent)
  - [`setOrigin`](#setorigin)
```
Now you won't need to do a Table of Contents for Action Creators. Since there is no overview text for Action Creators in each Duck API doc, the table of contents don't look right IMO and that's why I propose we hoist them up.